### ISSUE TITLE
Is there a way to upload diffs without keeping local backups?

### ISSUE THREAD

**Issue by bzon** [Altinity/clickhouse-backup#110]
A clickhouse server replica that performs the backup using this tool requires more significant start-up time (2-10 minutes). The disk space of this clickhouse replica is almost double of the other replicas within the same shard.

From what I understand, if we do not keep the local backups, the **upload with diff command** will have to download the diffs from remote storage which adds disk space.

In summary, I want to know how to do efficient delta backup. I want to delete local backups after the upload to remote storage. But I don't want to download the previous backups so that delta backup can be performed again.

**Comment by bzon** [comment]
I found the problem. We have TTL In our tables. The deleted partitions are deleted in `/var/lib/clickhouse/data` but not in `/var/lib/clickhouse/backup`.
